Parsely/wp-parsely
# API Improvements for REST-powered Admin page

## Description


Targeting `remove/jquery-dependency--try-react` branch in #239.  This is intended to be merged in there.

## Motivation and Context


So we can update settings over the REST API.

## How Has This Been Tested?



* Check out branch
* `npm i` to pull in the new dependency
* `npm run start` to build the updated client
* browse to settings page
  * settings should still pull from the API and populate the state tree
* click the submit button
  * settings should be sent to the new `POST` endpoint and interpreted there

The write endpoint will likely not succeed as-is...we need to make some changes to the client code to send the data appropriately to pass the validation.
